atlas/packages/app/src/shared/components/Searchbar/Searchbar.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Input } from './Searchbar.style'
+
+type SearchbarProps = {
+  value: string
+} & React.DetailedHTMLProps<React.HTMLAttributes<HTMLInputElement>, HTMLInputElement>
+const Searchbar: React.FC<SearchbarProps> = ({
+  placeholder,
+  onChange,
+  onFocus,
+  value,
+  onBlur,
+  onSubmit,
+  ...hmtlProps
+}) => {
+  return (
+    <Input
+      value={value}
+      placeholder={placeholder}
+      type="search"
+      autoSave="some_unique_value"
```

Comment:
What is that value?

Reply:
`autoSave` is a non standard attribute that's supposed to persist the input string between page refreshes, I had seen it and it looked useful, so I've added it, thinking about it, this is useless with React. So I'm removing it now.

https://www.wufoo.com/html5/autosave-attribute/

Reply:
@Gamaranto I think this isn't removed